src/messaging/interface/verifier/request.ts

```diff
@@ -0,0 +1,142 @@
+import { randomAsHex } from '@polkadot/util-crypto'
```

Comment:
Do we ever check if the CTypes here are valid CTypes?

Reply:
I added a test in the request function: [24ee6a0](https://github.com/KILTprotocol/kilt-extension-api/pull/24/commits/24ee6a0f98523f37885ebfb6d8cae9c93e5075db)